main/templates/main/project_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends "main/base_create.html" %}
+
+{% block create %}
+  <h2>Create a Project</h2>
+
+  <p>
+    Fill in the form below to create a new project and add it to the database.
+
+    *Note: Only members of the HoRSE group can create new projects.*
```

Comment:
Actually, only superusers. The `HoRSE` group is used specifically - and only for - notifications. 

```suggestion
```